testsuite/classlibrary/TestReadableNodeIDAllocator.sc

```diff
@@ -0,0 +1,85 @@
+TestReadableNodeIDAllocator : UnitTest {
+
+	// test proper creation from Server
+	test_readableNodeRange {
+		var s = Server(\testRNIA);
+		var prevClass = Server.nodeAllocClass;
+		Server.nodeAllocClass = ReadableNodeIDAllocator;
+		s.options.maxLogins = 1; s.newAllocators;
+		// this.assert(
```

Comment:
this commented out code: is it not better to move it out/remove?

Reply:
@adcxyz still waiting for info on this